bash_utils/config.sh

```diff
@@ -173,9 +182,8 @@ function shp_export {
 }
 
 
-#colp, facdb
-function fgdb_export {
-    urlparse ${BUILD_ENGINE}
+function fgdb_export_no_docker {
```

Comment:
`fgdb_export_no_docker` is defined twice. guessing the first one is the intended one?

Reply:
Oops. Yup